compute/gpu/troubleshooting/updating-gpu-instance-with-cloud-init.mdx

```diff
@@ -0,0 +1,34 @@
+---
+meta:
+  title: Fixing GPU issues after upgrading GPU Instances with cloud-init
+  description: This section provides information how to fix GPU issues after upgrading GPU Instances with cloud-init
+content:
+  h1: Fixing GPU issues after upgrading GPU Instances with cloud-init
+  paragraph:  This section provides information how to fix GPU issues after upgrading GPU Instances with cloud-init
+dates:
+  validation: 2022-09-19
+  posted: 2022-09-19
+---
+
+When running the `upgrade_package` command with `cloud-init` the nivida drivers may break and the GPU can become unavailable. This problem presists even after a manual reboot of the Instance and the following error displays: 
+
+```
+The GPU is not usable. The driver not loaded, with the following errors in the system journal
+
+[FAILED] Failed to start NVIDIA Persistence Daemon.
+```
+
+This error is caused by the way the `upgrade_package ` command is [handling the upgrade](https://cloudinit.readthedocs.io/en/latest/topics/examples.html#additional-apt-configuration-and-repositories) of the packages installed on the system. It runs `apt-get dist-upgrade` instead of `apt upgrade`.
```

Comment:
```suggestion
This error is caused by the way the `upgrade_package ` command [handles the upgrade](https://cloudinit.readthedocs.io/en/latest/topics/examples.html#additional-apt-configuration-and-repositories) of the packages installed on the system. It runs `apt-get dist-upgrade` instead of `apt upgrade`.
```